stubs/pycocotools/pycocotools/__init__.pyi

```diff
@@ -0,0 +1,3 @@
+from .coco import *
+from .cocoeval import *
+from .mask import *
```

Comment:
It looks like none of these `from .foo import *` imports exist at runtime, and that's causing most of the stubtest failures: at runtime `__init__.py` is just an empty file (other than `__author__`): https://github.com/ppwwyyxx/cocoapi/blob/71e284ef862300e4319aacd523a64c7f24750178/PythonAPI/pycocotools/__init__.py#L1